org.tenacityaudio.Tenacity.yaml

```diff
@@ -0,0 +1,271 @@
+    - -DRAPIDJSON_BUILD_EXAMPLES=OFF
+    - -DRAPIDJSON_BUILD_TESTS=OFF
+    - -DRAPIDJSON_BUILD_THIRDPARTY_GTEST=OFF
+    sources:
+      - type: archive
+        url: https://github.com/miloyip/rapidjson/archive/v1.1.0.tar.gz
+        sha256: bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e
+
+  - name: portaudio
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/PortAudio/portaudio.git
+        tag: v19.7.0
+        commit: 147dd722548358763a8b649b3e4b41dfffbcfbb6
+
+  - name: portsmf
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/tenacityteam/portsmf.git
+        tag: '238'
+        commit: f9191b5cf2fee63635e14328e5b235d859a81cb5
+
+  - name: portmidi-mixxxdj
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/mixxxdj/portmidi.git
+        tag: '236'
+        commit: fda97c3be2baf10438574d10bb2a6b8b4f1140c4
+
+  - name: soxr
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/chirlu/soxr.git
+        tag: 0.1.3
+        commit: 945b592b70470e29f917f4de89b4281fbbd540c0
+
+  - name: libsbsms
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/claytonotey/libsbsms.git
+        tag: '2.3.0'
+        commit: e99cd7e6c6367e476577be34d2fdbe2023904d7e
+
+  - name: vamp-plugin-sdk
+    sources:
+      - type: git
+        url: https://github.com/c4dm/vamp-plugin-sdk.git
+        tag: vamp-plugin-sdk-v2.10
+        commit: 67adfc2bf9486912a0fce5123cf54360ea2678bc
+    cleanup:
+      - /bin
+
+  - name: soundtouch
+    sources:
+      - type: git
+        url: https://gitlab.com/soundtouch/soundtouch.git
+        tag: '2.2'
+        commit: 762f56024b7ade81f6565903161dffec0ad4741e
+
+  - name: twolame
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/twolame/twolame-0.4.0.tar.gz
+        sha256: cc35424f6019a88c6f52570b63e1baf50f62963a3eac52a03a800bb070d7c87d
+
+  - name: tenacity
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
+      - -Dgtk=gtk+-3.0
+      - -Dglib=glib-2.0
+      - -DwxWidgets_INCLUDE_DIRS=/app/include/wx-3.1;/app/lib/wx/include/gtk3-unicode-3.1
+      - -DwxWidgets_CXX_FLAGS=-pthread
+      - -DwxWidgets_DEFINITIONS_GENERAL=WXUSINGDLL;__WXGTK__;_FILE_OFFSET_BITS=64
+      - -DwxWidgets_LIBRARIES=-lwx_gtk3u_xrc-3.1;-lwx_gtk3u_html-3.1;-lwx_gtk3u_qa-3.1;-lwx_gtk3u_core-3.1;-lwx_baseu_xml-3.1;-lwx_baseu_net-3.1;-lwx_baseu-3.1
+      - -Dlib_preference=system
+      - -Dobey_system_dependencies=On
+      - -Duse_wxwidgets=system
+      - -Duse_soxr=system
+      - -Duse_portaudio=system
+      - -Duse_portmixer=off
+      - -Duse_vamp=system
+      - -Duse_pa_jack=linked
+      - -Duse_soundtouch=system
+      - -Duse_twolame=system
+      - -Dhas_networking=no
```

Comment:
Please update for the changes from https://github.com/tenacityteam/tenacity/pull/228. You may need to set the WX_CONFIG environment variable to get CMake to find wxWidgets.

Reply:
Where would wxWidgets be installed? I just ran `find` inside the Tenacity container but I couldn't find it.

Reply:
I'm not that familiar with flatpak-builder to know but my guess would be `/app` is the default. You can explicitly set that by passing `-DCMAKE_INSTALL_PREFIX=/some/path` with any project that uses CMake.

Reply:
wx-config gets installed to CMAKE_INSTALL_PREFIX/bin/wx-config and you need to set the full path to the script in the environment variable.

Reply:
Ah, that's why I couldn't find it. We explicitly remove that directory https://github.com/TheEvilSkeleton/flatpaks/blob/9159b31769576adf9999fa74c0b8f6030089de99/org.tenacityaudio.Tenacity.yaml#L66

Reply:
@Be-ing do we use vcpkg or do we keep it that way?

Reply:
Also, is this good? https://github.com/flathub/flathub/pull/2444/commits/0cb65e3e13af9ae11064c1b3a15debc1dcda0f08

Reply:
Do not use vcpkg. It will not work with flatpak-builder currently. That could be made to work by adding the vcpkg executable to the runtime and running `vcpkg install --only-downloads` in the prepare step with network access. But there's no pressing need to do so as long as all the dependencies are built in the Flatpak manifest.